crates/superposition_types/src/database/schema.rs

```diff
@@ -646,6 +649,33 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    use diesel::sql_types::*;
+    use super::sql_types::OrgStatus;
+
+    organisation (id) {
+        #[max_length = 30]
+        id -> Varchar,
+        #[max_length = 10]
+        country_code -> Nullable<Varchar>,
+        #[max_length = 255]
+        contact_email -> Nullable<Varchar>,
+        #[max_length = 15]
+        contact_phone -> Nullable<Varchar>,
+        #[max_length = 255]
+        created_by -> Varchar,
+        #[max_length = 255]
+        admin_email -> Varchar,
+        status -> OrgStatus,
+        #[max_length = 100]
+        sector -> Nullable<Varchar>,
+        created_at -> Timestamp,
+        updated_at -> Timestamp,
+        #[max_length = 255]
+        updated_by -> Varchar,
+    }
+}
+
```

Comment:
@sauraww since this is inside superposition schema, it should look different
```
pub mod superposition {
    pub mod sql_types {
        #[derive(diesel::sql_types::SqlType)]
        #[diesel(postgres_type(name = "workspace_status", schema = "superposition"))]
        pub struct WorkspaceStatus;
    }

    diesel::table! {
        use diesel::sql_types::*;
        use super::sql_types::WorkspaceStatus;

        superposition.workspaces (organization_id, workspace_name) {
            organization_id -> Text,
            organization_name -> Text,
            workspace_name -> Text,
            workspace_schema_name -> Text,
            workspace_status -> WorkspaceStatus,
            workspace_admin_email -> Text,
            created_by -> Text,
            last_modified_by -> Text,
            last_modified_at -> Timestamp,
            created_at -> Timestamp,
            mandatory_dimensions -> Nullable<Array<Text>>,
        }
    }
}
```
This is what workspace looks like for the same create statement as yours.
You have to change diesel.toml to reflect this:
```
[print_schema.superposition]
file = "src/superposition_schema.rs"
patch_file = "schema.patch"
```